instance/tests/integration/integration_instance.py

```diff
@@ -439,7 +436,7 @@ def test_ansible_failure(self, git_checkout, git_working_dir):
         self.assertEqual(appserver.status, AppServerStatus.ConfigurationFailed)
         self.assertEqual(appserver.server.status, ServerStatus.Ready)
 
-    @skipIf(TEST_GROUP is not None and TEST_GROUP != '4', "Test not in test group.")
+    @skipIf(TEST_GROUP is not None and TEST_GROUP != '2', "Test not in test group.")
```

Comment:
With a single integration test running, do we need these decorators? If not, can you remove them?

Reply:
I'm keeping two test groups. One for the test that spawn an appserver and another for those that don't. This has the potential to make retrying the workflow faster if the 1st integration test group passes.